src/main/java/com/ty/workSpace_Management/dao/FloorDao.java

```diff
@@ -0,0 +1,53 @@
+package com.ty.workSpace_Management.dao;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import com.ty.workSpace_Management.entity.FloorEntity;
+import com.ty.workSpace_Management.repo.FloorRepo;
+
+@Repository
+public class FloorDao {
+
+	@Autowired
+	private FloorRepo repo;
+
+	public FloorEntity saveFloorEntity(FloorEntity floorEntity) {
+
+		return repo.save(floorEntity);
+
+	}
+
+	public FloorEntity getFloor(String FloorId) {
+		Optional<FloorEntity> floor = repo.findById(FloorId);
+		if (floor.isPresent()) {
+			return floor.get();
+		}
+		return null;
+	}
+
+	public FloorEntity updateFloorEntity(String floorId, FloorEntity floorEntity) {
+		FloorEntity floor = getFloor(floorId);
+		if (floor != null) {
+			floorEntity.setFloorId(floorId);
```

Comment:
pls write a code in such a way that check the user were providing the workspaces value ..if wont try to set old value

Reply:
i will be  doing this validation in the service layer